### PR TITLE
Blog post: JabRef & WinEdt

### PR DESCRIPTION
**Pull request by mlep**
For a release in about one week (not to have to many posts in a short time).

### CODE REVIEW

File: _posts/2016-04-25-WinEdtBothDirections.md [hunk 1]
```diff
@@ -3,17 +3,21 @@ title: JabRef & WinEdt: two-way communications
...
-Up to know, communication was only one-way: from JabRef to WinEdt.
+Up to now, communication was only one-way: from JabRef to WinEdt.
 This time is over: since April 2, 2016, WinEdt can also communicate with Jabref!
 
 Thanks to [a package contributed by Karl Koeller](http://www.winedt.org/config/menus/JabRef.html),
 JabRef can be launched from within WinEdt. 
 
+![FromWinEdtToJabRef](img/WinEdt-JabRef.png)
+
 _**You like it? Foster JabRef development with [Code](https://github.com/JabRef/jabref) or [Money](https://github.com/JabRef/jabref/wiki/Donations).**_
```
Comment (simonharrer) [inline _posts/2016-04-25-WinEdtBothDirections.md]: Please use the updated version of this line

**Comment by simonharrer** [comment]
:+1:

We should always push each blog post through the mailing list.

File: _posts/2016-04-25-WinEdtBothDirections.md (hunk 1 above)
Comment (mlep) [inline _posts/2016-04-25-WinEdtBothDirections.md]: Done! Thanks.

**Comment by koppor** [comment]
PRs should go into `gh-pages`. I fixed it here locally.

Site note: blog posts with a future date will be published at the future date and not earlier. - See http://www.fizerkhan.com/blog/posts/Working-with-upcoming-posts-in-Jekyll.html

**Comment by mlep** [comment]
@koppor It looks like the publication at future date does not work...

**Comment by matthiasgeiger** [comment]
It works - but not as expected:

The page has to built after the future date of the post - and this does not happen automatically. 
I just edited another post, and after this change your WinEdt also appears automatically.

@koppor Is there a way to force a jekyll built here at GitHub without pushing a commit to the repo? I found none... [Apart from this "hack"](http://stackoverflow.com/questions/24098792/how-to-force-github-pages-build)

**Comment by mlep** [comment]
@matthiasgeiger Thanks!

**Comment by koppor** [comment]
@matthiasgeiger I haven't been testing whether GitHub is smart enough to publish blog posts when they are ready. Seems, that GitHub isn't. There is no other possibility to trigger a build. A `git push -f` also works.

@mlep Sorry, I missed to write something, maybe you can do?

I still have on my agenda to change the URL of the blog to http://blog.jabref.org. This makes the maintainance on my side much more easy.

**Pull request merged**